packages/client/hmi-client/src/model-representation/mira/mira.ts

```diff
@@ -107,14 +112,27 @@ export const collapseParameters = (miraModel: MiraModel, miraTemplateParams: Mir
 			continue;
 		}
 
+		const nameContainsContext = _.intersection(tokens, contextValues).length > 0;
+		const hasNumericSuffixes =
+			tokens.length > 1 &&
+			tokens.every((d, idx) => {
```

Comment:
Why the `tokens.length > 1` test, `every` works on empty array as well. Or just to be sure?

Reply:
We want `x_123` to be true, but `x_abc` to be false and `x` to be false.